dali/base/dadfs.cpp

```diff
@@ -4767,19 +4772,26 @@ protected: friend class CDistributedFilePart;
         getModificationTime(dt);
         double fileAgeDays = difftime(time(nullptr), dt.getSimple())/(24*60*60);
         double sizeGB = getDiskSize(true, false) / ((double)1024 * 1024 * 1024);
+        const IPropertyTree *attrs = root->queryPropTree("Attr");
+        long long numDiskWrites = 0, numDiskReads;
```

Comment:
minor: long long is a bit ununusual.  We normally use __int64 or in this case possibly stats_type.

We should probably go through the entire code and use int64_t instead....

Reply:
numDiskReads is uninitialised.  Not sure why the compiler didn't catch it.

Reply:
vscode intellisense showed the return type as long long.  Probably, should have remembered this.